DEPLOYING.md

```diff
@@ -0,0 +1,27 @@
+# How to Deploy the package to NPM
+
+NPM registry at: https://www.npmjs.com/package/web-activities
+
+NPM login with credentials to web-activities package is required to publish.
+
+Once changes are made to the core library and have been approved, follow
+these steps to publish latest version to NPM:
+
+* Bump the version number in package.json such as: version: "1.0.20" -> "1.0.21"
+* Run the `'gulp activities-to-es6'` task to generate an intermediate file at
```

Comment:
We need to combine `gulp activities-to-es6` with `gulp dist`. We should only ever call one `gulp dist` for something like this. Further, no need to mention the intermediate files, just the end result.

Reply:
Updated the `gulp dist` task to include `activities-to-es6` and updated the doc.